python/interpret_community/common/constants.py

```diff
@@ -305,6 +305,7 @@ class Extension(object):
     """Provide constants for extensions to interpret package."""
 
     BLACKBOX = 'blackbox'
+    GLASSBOX = 'model'
```

Comment:
This is a bit confusing

Reply:
Is it because this is how it's implemented in `interpret`?

Reply:
indeed, that seems to be the name in interpret, not sure why, will follow up with them